scripts/run-e2e-tests.sh

```diff
@@ -100,6 +100,19 @@ function install_kind() {
   check_cmd "${KIND}"
 }
 
+function install_spark() {
+  check_cmd "wget"
+  wget -qO-  https://archive.apache.org/dist/spark/spark-3.3.1/spark-3.3.1-bin-hadoop3.tgz | tar xzvf - && chmod +x spark-3.3.1-bin-hadoop3 && sudo mv spark-3.3.1-bin-hadoop3 /opt/.
```

Comment:
If we can support pass spark version ENV to support different version of spark.

Reply:
+1 this sounds like a reasonable addition